atom/browser/native_window_views.h

```diff
@@ -18,6 +18,8 @@
 #include "atom/browser/ui/win/message_handler_delegate.h"
 #include "atom/browser/ui/win/taskbar_host.h"
 #include "base/win/scoped_gdi_object.h"
+#include "ui/base/win/accessibility_misc_utils.h"
```

Comment:
@kevinsawicki Without it, the next header barfs a thousand lines of COM nonsense